third_party/blink/web_tests/external/wpt/resources/chromium/cobalt/fake-h5vcc-system-impl.js

```diff
@@ -0,0 +1,56 @@
+import {H5vccSystem, H5vccSystemReceiver} from '/gen/cobalt/browser/h5vcc_system/public/mojom/h5vcc_system.mojom.m.js';
+
+// Implementation of h5vcc_system.mojom.H5vccSystem.
+class FakeH5vccSystemImpl {
+  constructor() {
+    console.log('FakeH5vccSystemImpl constructor');
+    this.interceptor_ =
+        new MojoInterfaceInterceptor(H5vccSystem.$interfaceName);
+    this.interceptor_.oninterfacerequest = e => this.bind(e.handle);
+    this.receiver_ = new H5vccSystemReceiver(this);
+    this.stub_result_ = '';
+  }
+
+  start() {
+    console.log('FakeH5vccSystemImpl start');
+    this.interceptor_.start();
```

Comment:
I'm comparing this implementation to [1] and there're a few differences, e.g. interceptor_.start() is run in the constructor() in [1]. reset() calls interceptor_.stop() etc. Maybe we can make this class similar to [1] which is known to work : )


[1] https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/web_tests/external/wpt/resources/chromium/mock-barcodedetection.js;l=7-56